modules/ROOT/pages/upgrade-index.adoc

```diff
@@ -10,7 +10,7 @@ Anypoint Runtime Fabric is a software appliance that combines packages that work
 +
 Appliance upgrades affect platform-layer components, which include Docker, Kubernetes, and the Ops Center. These platform-layer components are installed on virtual machines (VMs) or bare-metal servers that act as nodes on the cluster.
 +
-Appliance upgrades are applied using the `rtfctl` command-line utility.
+Appliance upgrades are applied using the `rtfctl appliance upgrade` command-line utility.
```

Comment:
`rtfctl` is command-line utility, so earlier wording seems right.
`rtfctl appliance upgrade` is the command-line option for upgrading Runtime Fabric appliance. This can't be worded as command-line utility.